app/screens/PostCreate.tsx

```diff
@@ -32,7 +32,7 @@ export default function (props: any) {
   const [image, setImage] = useState<ImageOrVideo | undefined>();
 
   const insert = useReactGraphql(HasuraConfig.posts).useInsert({
-    initialVariables: {userId, groupId},
+    initialVariables: {groupId},
```

Comment:
In hasura permissions the userId has a default based on the JWT.  So including it here as well causes a failure.